components/eam/chem_proc/inputs/pp_chemUCI_linozv3_mam5_vbs.in

```diff
@@ -208,7 +208,9 @@ CHEMISTRY
 [lhno4]     HO2NO2 + OH -> NO2 + H2O + O2                  ; 1.300E-12, 380
 [lhno3]     HNO3 + OH -> NO3 + H2O                         ; 2.400E-14, 460
 *   fix needed to add this bimolec rate, missing from the one below
-[uci6] HNO3 + OH -> NO3 + H2O
+[uci6] HNO3 + OH -> NO3 + H2O                              ; 2.400E-14, 460
+* uci6 coefficients assigned here are to avoid debug issue
+* the real coefficients will calculated in the model
```

Comment:
I suggest we make the comments clearer and change to "We need these uci6 coefficients to be non-blank to avoid issues with debug-build. Model actually uses customized coefficients calculated in llnl_O1D_to_2OH_adj.F90".

Reply:
changed